OpenAPIService.Test/OpenAPIServiceShould.cs

```diff
@@ -199,11 +199,12 @@ public void ReturnOpenApiDocumentInCreateFilteredDocumentWhenValidArgumentsAreSp
         }
 
         [Theory]
-        [InlineData(OpenApiStyle.Plain, "/users/{user-id}")]
-        [InlineData(OpenApiStyle.GEAutocomplete, "/users/{user-id}")]
-        [InlineData(OpenApiStyle.PowerShell, "/administrativeUnits/{administrativeUnit-id}/microsoft.graph.restore")]
-        [InlineData(OpenApiStyle.PowerPlatform, "/administrativeUnits/{administrativeUnit-id}/microsoft.graph.restore")]
-        public void ReturnOpenApiDocumentInApplyStyleForAllOpenApiStyles(OpenApiStyle style, string url)
+        [InlineData(OpenApiStyle.Plain, "/users/{user-id}", OperationType.Get)]
+        [InlineData(OpenApiStyle.GEAutocomplete, "/users/{user-id}", OperationType.Get)]
+        [InlineData(OpenApiStyle.PowerPlatform, "/administrativeUnits/{administrativeUnit-id}/microsoft.graph.restore", OperationType.Post)]
+        [InlineData(OpenApiStyle.PowerShell, "/administrativeUnits/{administrativeUnit-id}/microsoft.graph.restore", OperationType.Post)]
+        [InlineData(OpenApiStyle.PowerShell, "/users/{user-id}", OperationType.Patch)]
```

Comment:
Add the following test scenario to better validate the change. You will notice that both `PATCH` and `PUT` operations are renamed to `*_set*`.
```suggestion
        [InlineData(OpenApiStyle.PowerShell, "/users/{user-id}", OperationType.Patch)]
        [InlineData(OpenApiStyle.PowerShell, "/applications/{application-id}/logo", OperationType.Put)]
```